dev/otel-agent-config.yaml

```diff
@@ -1,4 +1,8 @@
 receivers:
+  jaeger:
```

Comment:
which system needs jaeger?

Reply:
the rust systems (stablesats / bria) are still talking to the jaeger endpoints. An easy 'maintenance' task to move them to otlp. cc @thevaibhav-dixit